Azure/azure-cli
# {Core} Add storage sync endpoint suffix

**Description**  

This PR adds the storage sync endpoint suffix for AzureCloud and AzureUSGovernment. 

It will use the property name `storageSyncEndpointSuffix` to find the suffix in ARM cloud discovery, and it will use the suffix of AzureCloud as default if couldn't find it.

**Testing Guide**  

Run the command below, the PR will make the suffix `storageSyncEndpoint` appear.
`az cloud show -n AzureCloud`
